### PR TITLE
User updating

### PR DESCRIPTION
**Pull request by nazar110**

### CODE REVIEW

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 1]
```diff
@@ -314,11 +324,22 @@ private void DeleteAccountButton_Click(object sender, RoutedEventArgs e)
...
         }
+
+        private void SavaChangeButton_Click(object sender, RoutedEventArgs e)
+        {
+            this.ChangesGrid.Visibility = Visibility.Hidden;
+        }
+
         private void LogOutButton_Click(object sender, RoutedEventArgs e)
         {
+>>>>>>> cfcf0adac9e9034abafbf8911e36d40a85ac20a4
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 2]
```diff
@@ -314,11 +324,22 @@ private void DeleteAccountButton_Click(object sender, RoutedEventArgs e)
             UserEditor userEditor = new UserEditor();
             List<string> parameters = new List<string>() { ChangeName.Text, ChangeSurname.Text, ChangeEmail.Text, ChangePassword.Text, ChangeConfirmPassword.Text };
             userEditor.UpdateUser(parameters);
-            //userEditor.error != "" then show window
+            ChangesGrid.Visibility = Visibility.Hidden;
+        }
+
+        public void LogOutButton_Click(object sender, RoutedEventArgs e) {
+=======
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 3]
```diff
@@ -306,6 +315,7 @@ private void DeleteAccountButton_Click(object sender, RoutedEventArgs e)
         {
             YNoteWPF.PL.AdditionalWindow.ConfirmDelete confirmdeleteWindow = new AdditionalWindow.ConfirmDelete();
             confirmdeleteWindow.Show();
+<<<<<<< HEAD
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 4]
```diff
@@ -28,7 +31,13 @@ public partial class MainWindow : Window {
         private double firstXPos;
         private double firstYPos;
         private object movingObject;
+
+<<<<<<< HEAD
+        //UserEditor userEditor = new UserEditor();
+        
+=======
         public UserDTO User { get; set; }
+>>>>>>> cfcf0adac9e9034abafbf8911e36d40a85ac20a4
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 5]
```diff
@@ -28,7 +31,13 @@ public partial class MainWindow : Window {
         private double firstXPos;
         private double firstYPos;
         private object movingObject;
+
+<<<<<<< HEAD
+        //UserEditor userEditor = new UserEditor();
+        
+=======
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 6]
```diff
@@ -28,7 +31,13 @@ public partial class MainWindow : Window {
         private double firstXPos;
         private double firstYPos;
         private object movingObject;
+
+<<<<<<< HEAD
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 7]
```diff
@@ -15,8 +15,11 @@
 using System.Windows.Shapes;
 using MaterialDesignThemes;
 using System.IO;
+<<<<<<< HEAD
 using YNoteWPF.BLL.UserOperations;
+=======
 using YNoteWPF.BLL.Data.Models;
+>>>>>>> cfcf0adac9e9034abafbf8911e36d40a85ac20a4
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 8]
```diff
@@ -15,8 +15,11 @@
 using System.Windows.Shapes;
 using MaterialDesignThemes;
 using System.IO;
+<<<<<<< HEAD
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

File: YNoteWPF.PL/MainWindow.xaml.cs [hunk 9]
```diff
@@ -15,8 +15,11 @@
...
 using YNoteWPF.BLL.UserOperations;
+=======
```
Comment (FoxLviv) [inline YNoteWPF.PL/MainWindow.xaml.cs]: Fix conflicts

**Review by FoxLviv — CHANGES_REQUESTED** [review]

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 1 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 2 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 3 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 4 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 5 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 6 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 7 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 8 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

File: YNoteWPF.PL/MainWindow.xaml.cs (hunk 9 above)
Comment (nazar110) [reply YNoteWPF.PL/MainWindow.xaml.cs]: Done

**Comment by FoxLviv** [comment]
reviewed

**Pull request merged**